google-cloud-speech/lib/google/cloud/speech/project.rb

```diff
@@ -626,7 +626,6 @@ def operation id
         def audio_config encoding: nil, language: nil, sample_rate: nil,
                          max_alternatives: nil, profanity_filter: nil,
                          phrases: nil, words: nil
-          contexts = nil
```

Comment:
While this doesn’t change functionality, it will issue a warning when `contexts` is used without first being initialized.